daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

```diff
@@ -58,6 +70,41 @@ To terminate your workflow with an ID `12345678`, run:
 POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/terminate
 ```
 
+### Raise an event
+
+For your workflow to create "wait for external event" tasks that:
+
+1. Subscribe to external events.
+1. Await those tasks to block execution until the event is received.
```

Comment:
Nit: Should be a 2 instead of a 1

Reply:
In markdown, using `1.` for all numbers in a numbered list translates to `1.`, `2.`, `3.`, etc. once published, without you having to remember to renumber the list if you add steps later.